src/System.Diagnostics.Process/src/Resources/Strings.resx

```diff
@@ -243,8 +243,8 @@
   <data name="ProcessorAffinityNotSupported" xml:space="preserve">
     <value>Processor affinity for processes or threads is not supported on this platform.</value>
   </data>
-  <data name="ProcessStartIdentityNotSupported" xml:space="preserve">
-    <value>Starting a process with a different identity is not supported on this platform.</value>
+  <data name="ProcessStartFeaturesNotSupported" xml:space="preserve">
+    <value>{0} field(s) is not supported by "run as another user" feature on this platform.</value>
```

Comment:
I still think it makes sense to split this into 2 different exception messages, one for the ProcessStartInfo properties' get/set methods and one for the Process.Start overloads.

I like the idea of passing in the name of which property is not supported in the ProcessStartInfo class.